libvast/src/detail/process.cpp

```diff
@@ -63,4 +130,15 @@ caf::expected<path> objectpath(const void* addr) {
   return objectpath_static();
 }
 
+caf::settings get_status() {
```

Comment:
```suggestion
caf::expected<caf::settings> get_status() {
```

Can we make this return an expected settings, such that downstream can handle the error in a unique way, and that the third path can return `make_error(ec::unimplemented)`?

Reply:
In this particular case I would rather keep the return type as `caf::settings` for simplicity, the only calling site is in `node.cpp`, and there we have nothing else to do than discard an error.

Reply:
Fair enough.